examples/siro_sandbox/controllers/gui_controller.py

```diff
@@ -195,9 +197,21 @@ def __init__(
         self._hint_distance_multiplier = None
         self._hint_grasp_obj_idx = None
         self._hint_drop_pos = None
+        self._hint_throw_vel = None
+        self._hint_reach_pos = None
+        self._is_picking = None
+        self._iter_pose = 0
         self._cam_yaw = 0
         self._saved_object_rotation = None
         self._recorder = recorder
+        self._obj_to_grasp = None
+        self._thrown_object_collision_group = CollisionGroups.UserGroup7
+        self._last_object_thrown_info = None
+
+        self.ind = 0
+
+    def set_humanoid_controller(self, humanoid_controller):
+        self._humanoid_controller = humanoid_controller
 
     def get_articulated_agent(self):
         return self._env._sim.agents_mgr[self._agent_idx].articulated_agent
```

Comment:
Compared to [this version](https://github.com/facebookresearch/habitat-lab/blob/eundersander/SIRo_hitl_Sep5_demo_refactored/examples/siro_sandbox/controllers/gui_controller.py#L232), you've removed a lot of code that runs in on_environment_reset. Is that safe?

Reply:
Good catch. This specific code was not ported over.